source/Popup/Views/Contacts/components/AddContact.jsx

```diff
@@ -26,6 +26,8 @@ const AddContact = ({ handleAddContact, contacts }) => {
     setId(value.trim());
   };
 
+  const handleAddContact = () => addContact({ name, id, image: getRandomEmoji() });
```

Comment:
+1 like this approach

Reply:
Yeah -- that's a great one
I've noticed that some component's props are called `handle(FunctionName)`
Let's keep the approach suggested by React:

1. Functions inside the components called `handle(FunctionName)`
2. Props, that are accepting functions called `(functionName)`